tests/comments.test.js

```diff
@@ -0,0 +1,253 @@
+const request = require("supertest");
+const initApp = require("../server.js");
+const mongoose = require("mongoose");
+const postsModel = require("../models/posts_model");
+const commentsModel = require("../models/comments_model");
+
+let app;
+let post;
+beforeAll(async () => {
+  app = await initApp();
+  await postsModel.deleteMany();
+  post = await postsModel.create({ content: "Test post", sender: "amitinbar" });
+});
+
+beforeEach(async () => {
+  await commentsModel.deleteMany();
+});
+
+afterAll(async () => {
+  mongoose.connection.close();
+});
+
+describe("POST /comments", () => {
+  it("should create new comment", async () => {
+    const content = "This is my first comment!";
+    const sender = "amitinbar";
+    const response = await request(app)
+      .post(`/comments?post_id=${post._id}`)
+      .send({
+        content,
+        sender,
+      });
+
+    expect(response.statusCode).toBe(200);
+    expect(response.body).toHaveProperty("_id");
+    expect(response.body.content).toBe(content);
+    expect(response.body.sender).toBe(sender);
+    expect(response.body.postID).toBe(post._id.toString());
+  });
+
+  it.each([{ content: "No sender" }, { sender: "No content" }, {}])(
```

Comment:
I think we will handle this in client side and won't let the user send an empty comment/
we will recieve the sender from the token